.changeset/long-bottles-brush.md

```diff
@@ -0,0 +1,23 @@
+---
+"@comet/cms-api": major
+---
+
+Rename DateFilter to DateTimeFilter
+
+This leaves room for a future DateFilter that only filters by date, not time.
+
+## Upgrading
```

Comment:
```suggestion
**Upgrading**
```

Changesets shouldn't have headings since those are set in the release (## for package name, ### for change type)

Reply:
09d0b8e2dff80462070e7802c6ab8dfa0cc8e5f4